bip-0009/states.dot

```diff
@@ -0,0 +1,29 @@
+digraph {
+  rankdir=TD;
+
+  node [style="rounded,filled,bold", shape=box, fixedsize=true, width=1.5, fontname="Arial"];
+
+  edge [weight = 100];
+  "DEFINED" -> "STARTED" [label="Always"];
```

Comment:
This is wrong, too. The correct state diagram is at https://github.com/bitcoin/bitcoin/pull/21377#issuecomment-812023131

Reply:
I took over most of the diagram, but left out the `DELAYED` pseudo-state.

Reply:
https://github.com/bitcoin/bitcoin/pull/21377#issuecomment-814768715 is the current state diagram

Reply:
I think I was only missing `height + 1`,so added that. Also expanded the text a bit.